### PR TITLE
Added boolean fields

### PR DESCRIPTION
**Pull request by pmelab**

### CODE REVIEW

File: modules/graphql_boolean/graphql_boolean.info.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+name: GraphQL Link
```
Comment (Xilis) [inline modules/graphql_boolean/graphql_boolean.info.yml]: Change name to GraphQL Boolean

File: modules/graphql_boolean/graphql_boolean.info.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+type: module
+description: 'Retrieve distinct properties from link fields.'
```
Comment (Xilis) [inline modules/graphql_boolean/graphql_boolean.info.yml]: Description could read something like: 'Retrieve boolean values from boolean fields.'

**Review by Xilis — CHANGES_REQUESTED** [review]
.info changes for module name/description

**Comment by fubhy** [comment]
Thanks for the review @Xilis.

**Comment by Xilis** [comment]
Just noticed you made the change just before I made a new PR for it (#97). Just ignore/close that one.

**Pull request merged**